docs/content/v0.20/contributor.md

```diff
@@ -59,6 +59,12 @@ X ArgoCD CLI
   how to install: https://argo-cd.readthedocs.io/en/stable/cli_installation/
 ```
 
+## Unit testing
+

```

Comment:
Do we want to add a line of description (e.g., In order to run the unit tests ......)

Reply:
done